src/index.ts

```diff
@@ -78,4 +85,9 @@ export = class OutputDcatUs11 {
       ),
     };
   }
+
+  private getVersion(reqPath: string) {
+    const version = reqPath.substring(reqPath.lastIndexOf('/') + 1);
+    return ['1.1', '3.0'].includes(version) ? version : '1.1';
```

Comment:
Should we throw an error if the version is something other that `1.1` or `3.0`?

Reply:
The plugin can be only served with routes `/dcat-us/3.0`, `/dcat-us/1.1` and `/dcat-us/data.json`. Without these routes, plugin won't work. So, path will always have either version number or `data.json`. In case of `data.json`, we should return 1.1 version number as it is primary one.